.circleci/config.yml

```diff
@@ -229,6 +229,7 @@ workflows:
               ignore:
                 - master
                 - bulk
+                - /bulk.*/
```

Comment:
This is kind of hacky, but I did it because master is hard-coded in this yml file as the root branch and so building this would try to build everything from bulk.

Azure has the option to exclude PRs with the target of bulk so I did that on the bulk branch itself before branching this..